schema/game_author.rb

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS 'authors' (
+  'id' INTEGER PRIMARY KEY AUTOINCREMENT,
+  'first_name' TEXT NOT NULL,
+  'last_name' TEXT NOT NULL,
+  'publish_date' DATE NOT NULL,
+  'archived' BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
These are defined in the item class. So no need to add them to the authors table.